src/pages/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: React.FC<Props> = ({
+  tabs,
+}) => {
```

Comment:
```suggestion
export const TabsPage: React.FC = () => {
```
Props are useless for your component.